user/deployment/biicode.md

```diff
@@ -0,0 +1,146 @@
+---
+title: biicode deployment
+layout: en
+permalink: biicode/
```

Comment:
We've made changes to permalinks recently, so that this needs to be `/usr/deployment/biicode/`.